.forge-snapshots/AsyncVault_requestDeposit.snap

```diff
@@ -1 +1 @@
-387182
\ No newline at end of file
+418500
```

Comment:
I don't understand why these are going up so much

Reply:
Are you locally using an the stable foundry version?

Reply:
I can confirm the issue is NOT related to the foundry usage. I got the same values locally 🤔

Reply:
I think there were already not updated in previous PR, because I got the same without your commit

Reply:
They seem to be outdated from previous commits, weird!